content/_layouts/base.njk

```diff
@@ -75,7 +75,6 @@
     {% include "site/alert.njk" %}
     {% include "site/banner.njk" %}
     {% include 'page/base.njk' %}
-    {% include 'site/cta.njk' %}
```

Comment:
I think this removes the only case where we were using the `workshop` cta before (https://www.oddbird.net/tools/). If that's intentional, should we remove the `cta/workshop.njk` file entirely, as well as the ability to define a custom cta (i.e. `cta_slug`)?